### PR TITLE
fix color completition

### PR DESCRIPTION
**Pull request by edusantana**
see #2835

**Short release notes**: Fix bash completion
**Longer description**: Now you can call kdb <TAB><TAB> and it will return

```
check          fstab          import         mount          shell
complete       get            info           mv             smount
convert        getmeta        list           remount        spec-mount
cp             global-mount   list-commands  rm             test
editor         global-umount  list-tools     rmmeta         umount
export         gmount         ls             set            vset
file           gumount        lsmeta         setmeta        
find           help           merge          sget
```

## Basics

Check relevant points but **please do not remove entries**.
Do not describe the purpose of this PR in the PR description but:

- [ ] Short descriptions should be in the release notes (added as entry in
      `doc/news/_preparation_next_release.md` which contains `_(my name)_`)
      **Fix Please always add something to the the release notes.**
- [ ] Longer descriptions should be in documentation or in design decisions.
- [ ] Describe details of how you changed the code in commit messages
      (first line should have `module: short statement` syntax)
- [ ] References to issues, e.g. `close #X`, should be in the commit messages.

## Checklist

Check relevant points but **please do not remove entries**.
For docu fixes, spell checking, and similar none of these points below
need to be checked.

- [ ] I added unit tests
- [X] I ran all tests locally and everything went fine
- [ ] affected documentation is fixed
- [ ] I added code comments, logging, and assertions (see [Coding Guidelines](https://master.libelektra.org/doc/CODING.md))
- [ ] meta data is updated (e.g. README.md of plugins and [METADATA.ini](https://master.libelektra.org/doc/METADATA.ini))

## Review

Reviewers will usually check the following:

- [ ] Code is conforming to [our Coding Guidelines](https://master.libelektra.org/doc/CODING.md)
- [ ] APIs are conforming to [our Design Guidelines](https://master.libelektra.org/doc/DESIGN.md)
- [ ] Code is consistent to [our Design Decisions](https://master.libelektra.org/doc/decisions)
- [ ] Documentation is concise and good to read
- [ ] Examples are well chosen and understandable

## Labels

- Add the "work in progress" label if you do not want the PR to be reviewed yet.
- Add the "ready to merge" label **if the build server is happy** and also you
  say that everything is ready to be merged.

### CODE REVIEW

**Comment by markus2330** [comment]
Thank you for your contribution!

Please, as stated by the build server, write a line in doc/news/_preparation_next_release.md to make sure your contribution will be mentioned in the next release notes.

Furthermore, there was a formatting problem (we use automatic code formatters) which is fixed in #2837.

File: scripts/kdb-bash-completion [hunk 1]
```diff
@@ -23,7 +23,7 @@ _kdb() {
 	# only kdb was entered yet, print a list of available commands
 	if [[ $COMP_CWORD -eq 1 ]]; then
 		local IFS=$'\n'
-		local commands=($(${kdbpath} 2> /dev/null | sed -e '0,/^Known commands are/d' | awk '{print $1}'))
+		local commands=($(${kdbpath} 2> /dev/null | sed -e '0,/^Known commands are/d' | awk '{print $1}' | sed -r "s/\x1B\[([0-9]{1,3}((;[0-9]{1,3})*)?)?[m|K]//g" ))
```
Comment (sanssecours) [inline scripts/kdb-bash-completion]: I think you can use the `kdb` subcommand [`list-commands`](https://www.libelektra.org/manpages/kdb-list-commands) to simplify this code: 

```suggestion
		local commands=($(${kdbpath} list-commands))
```

.

**Comment by edusantana** [comment]
@sanssecours that's even better!

**Comment by markus2330** [comment]
Thank you for your contribution! We hope to see more from you soon!

**Pull request merged**